tests/host_tools/network.py

```diff
@@ -95,7 +94,7 @@ def _init_connection(self):
         if ecode != 0:
             raise ConnectionError
 
-    def run(self, cmd_string, timeout=None):
+    def run(self, cmd_string, timeout=None, *, check_output=False):
```

Comment:
I'd rather try to mimic the `subprocess` module here. So if we want it to raise we call `check_output`, otherwise we call `run`. 

That would involve converting all current uses of run to check_output, so maybe that ship has sailed.

Reply:
Ok I see that you did that at the end of the PR :)